af3_mmseqs2/alphafold3.py

```diff
@@ -118,44 +120,52 @@ def main():
     with open(args.input_json, "r") as f:
         af3_json = json.load(f)
 
-    if args.mmseqs2:
-        af3_json = add_msa_to_json(
-            input_json=args.input_json,
-            templates=args.templates,
-            num_templates=args.num_templates,
-            custom_template=args.custom_template,
-            custom_template_chain=args.custom_template_chain,
-            target_id=args.target_id,
-            af3_json=af3_json,
-            output_json=args.output_json,
-            to_file=True,
-        )
-
-        output_json = (
-            args.input_json.replace(".json", "_mmseqs.json")
-            if args.output_json is None
-            else args.output_json
-        )
-    else:
-        output_json = args.input_json
-
-    args.chai_1 = False
-    if not args.alphafold3 and not args.boltz1 and not args.chai_1:  #
-        args.alphafold3 = True
-
-    if args.alphafold3:
-        run_alphafold3(
-            input_json=output_json,
-            output_dir=args.output_dir,
-            model_params=args.model_params,
-            database_dir=args.database_dir,
-        )
-    if args.boltz1:
-        run_boltz(
-            input_json=output_json,
-            output_dir=args.output_dir,
-            save_input=args.save_input,
-        )
+    with tempfile.TemporaryDirectory() as temp_dir_str:
```

Comment:
What's tempdir being used for here? Just trying to follow the logic

Reply:
The temp direcotry is for storing the intermediate json if args.output json wasn't specificied. If it was, we keep it in the output directory. If it isn't we save it the tmp dir to use but then it gets removed once the job is complete